content/post/2023/captioning-tools-extravaganza.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Captioning Tools Extravaganza!"
+date: 2023-10-09
+---
+
+So, you've found yourself attending or speaking at a conference that's not able to provide better accessibility tools. How do you navigate this?
```

Comment:
I think the intro needs to make it clearer who the audience is. While reading the intro, I thought the audience was deaf/hoh individuals, but while reading the blog post, I felt it was educating conference organizers & co. I believe you want to share your experience with both audiences, right?

Maybe something along these lines:

> In this blog post, I'll share my experience with various captioning tools. I am deaf, not fluent in American Sign Language, and rely heavily on captioning software for work and beyond. By sharing my experience, I hope to help employers, conference organizers, live streamers, or whoever might want to improve accessibility through captions. I also want to share my experience with my deaf and hard of hearing peers who are researching captioning tools for themselves.

Reply:
That is a great point, thanks! Let me give some thought to how I want to phrase that... the audience for the post _is_ rather wide. :)